pkg/packagekit/updates.jsx

```diff
@@ -665,8 +665,8 @@ class OsUpdates extends React.Component {
                     if (role !== PK.Enum.ROLE_UPDATE_PACKAGES)
                         return;
                     // data looks like:
-                    // downloading	bash-completion;1:2.6-1.fc26;noarch;updates-testing
-                    // updating	bash-completion;1:2.6-1.fc26;noarch;updates-testing
+                    // downloading\tbash-completion;1:2.6-1.fc26;noarch;updates-testing
```

Comment:
With this batch updated (multiple rules at once), it's hard to find which rules introduced that.
I think we should avoid updated inside comments

Reply:
This is why this PR has one commit per rule. This one is the "no-tabs" one.